.github/workflows/validate-jballerina-nightly.yml

```diff
@@ -0,0 +1,37 @@
+name: Validate on jBallerina nightly
+
+on:
+  schedule:
+    - cron: '0 */12 * * *'
+  workflow_dispatch:
+  
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install LLVM
+        run: |
+          sudo apt-get install llvm-11
+      - name: Download jBallerina
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo "LATEST_RUN_ID = $(gh run list --workflow main.yml  --repo ballerina-platform/ballerina-distribution | grep success | head -n 1)" >> $GITHUB_ENV
+          gh run download ${{ env.LATEST_RUN_ID }} -D "$HOME" --name "Linux installer ZIP" --repo ballerina-platform/ballerina-distribution
+          unzip "$HOME/ballerina-linux-swan-lake-beta3-SNAPSHOT.zip" -d "$HOME"
```

Comment:
lets use a wildcard here `ballerina-*.zip`

Reply:
acutaly it might be better to find the correct path and keep it in a variable since we need it for `$GITHUB_PATH`